src/frontend/src/common/analytics/eventTypes.ts

```diff
@@ -169,6 +169,10 @@ export type AnalyticsSamplesDownloadFile = {
   sample_public_ids: JsonString;
   // User downloaded info on consensus genome (FASTA) for these samples
   includes_consensus_genome: boolean;
+  // User downloaded genbank template for these samples
```

Comment:
@vincent-czi would like to make sure this is an ok change/what I should do

Reply:
Perfect: this change and the other one to pass te data looks great!